server-common/src/main/java/org/apache/kafka/server/util/FutureUtils.java

```diff
@@ -103,4 +104,12 @@ public static <T> CompletableFuture<T> failedFuture(Throwable ex) {
         future.completeExceptionally(ex);
         return future;
     }
+
+    public static <T> void drainFutures(
```

Comment:
I suppose that we could remove this now, isn't it?

Reply:
done.